src/lib/util.js

```diff
@@ -497,11 +497,50 @@ function getShortname(spec) {
 }
 
 
+/**
+ * Returns true when the given spec is is the latest level of that spec in the
+ * given list of specifications. Note the code handles the special case of the
+ * CSS2 and CSS22 specs, and assumes that URLs that don't end with a level
+ * number are at level 1 (this does not work for CSS specs whose URLs still
+ * follow the old `css3-` pattern, but we're only interested in comparing
+ * with more recent levels in that case, so it does not matter)
+ *
+ * The function ignores delta specs, which are never considered to be the last
+ * level.
+ *
+ * @function
+ * @public
+ * @param {Object} spec Spec to check
+ * @param {Array(Object)} list List of specs (may include the spec to check)
+ * @return {Boolean} true if the spec is the latest level to consider.
+ */
+function isLatestLevel(spec, list) {
```

Comment:
This looks like it would be pretty easy to add tests for in tests/util.js, if you'd like.

Reply:
Oops, indeed!

Reply:
Reaction was for previous comment actually. I'll see what I can easily do. That logic is going to change soon as the information about level is going to come from browser-specs.